PocketShop/ViewModels/CustomerViewModel.swift

```diff
@@ -85,20 +85,66 @@ final class CustomerViewModel: ObservableObject {
 
         let orderProductsGroups = Dictionary(grouping: orderProducts, by: { $0.shopId })
 
+        // If it does not has coupon, we can see it as if it has applied coupon
+        var hasAppliedCoupon: Bool = coupon == nil ? true : false
```

Comment:
I think this works as well
```suggestion
        var hasAppliedCoupon: Bool = coupon == nil
```

Reply:
Right, ok